docs/data/apis/api-providers.mdx

```diff
@@ -13,24 +13,27 @@ Multiple infrastructure providers have made Stellar RPC and Horizon services ava
 
 These providers allow access to the Futurenet, Testnet and Mainnet network.
 
-| Provider | Futurenet | Testnet | Mainnet | Horizon | RPC |
-| --- | --- | --- | --- | --- | --- |
-| [Blockdaemon\*†](https://www.blockdaemon.com/apply/soroban) | ❌ | ✅ | ✅ | ✅ | ✅ |
-| [Validation Cloud\*†](https://app.validationcloud.io/) | ❌ | ✅ | ✅ | ✅ | ✅ |
-| [QuickNode\*](https://www.quicknode.com/docs/stellar) | ❌ | ✅ | ✅ | ✅ | ✅ |
-| [NowNodes](https://nownodes.io/nodes/stellar-xlm) | ✅ | ✅ | ✅ | ❌ | ✅ |
-| [Gateway](https://gateway.fm/public-rpc/) | ❌ | ✅ | ✅ | ❌ | ✅ |
-| [Ankr](https://www.ankr.com/rpc/advanced-api/) | ❌ | ✅ | ✅ | ✅ | ✅ |
-| [Infstones](https://infstones.com/) | ❌ | ❌ | ✅ | ❌ | ✅ |
-| [Obsrvr](https://www.withObsrvr.com/) | ❌ | ✅ | ✅ | ✅ | ✅ |
-| [GetBlock](https://getblock.io/nodes/xlm/) | ❌ | ❌ | ✅ | ❌ | ✅ |
-| [Nodies](https://nodies.org) | ❌ | ✅ | ✅ | ✅ | ✅ |
-| [OnFinality](https://onfinality.io/networks/stellar) | ❌ | ❌ | ✅ | ❌ | ✅ |
+| Provider | Futurenet | Testnet | Mainnet | Horizon | RPC | RPC Historical Access ** |
+| --- | --- | --- | --- | --- | --- | --- |
+| [Blockdaemon\*†](https://www.blockdaemon.com/apply/soroban) | ❌ | ✅ | ✅ | ✅ | ✅ | ❌ |
+| [Validation Cloud\*†](https://app.validationcloud.io/) | ❌ | ✅ | ✅ | ✅ | ✅ | ❌ |
+| [QuickNode\*](https://www.quicknode.com/docs/stellar) | ❌ | ✅ | ✅ | ✅ | ✅ | ❌ |
+| [NowNodes](https://nownodes.io/nodes/stellar-xlm) | ✅ | ✅ | ✅ | ❌ | ✅ | ❌ |
+| [Gateway](https://gateway.fm/public-rpc/) | ❌ | ✅ | ✅ | ❌ | ✅ | ❌ |
+| [Ankr](https://www.ankr.com/rpc/advanced-api/) | ❌ | ✅ | ✅ | ✅ | ✅ | ❌ |
+| [Infstones](https://infstones.com/) | ❌ | ❌ | ✅ | ❌ | ✅ | ❌ |
+| [Obsrvr](https://www.withObsrvr.com/) | ❌ | ✅ | ✅ | ✅ | ✅ | ✅ |
+| [GetBlock](https://getblock.io/nodes/xlm/) | ❌ | ❌ | ✅ | ❌ | ✅ | ❌ |
+| [Nodies](https://nodies.org) | ❌ | ✅ | ✅ | ✅ | ✅ | ❌ |
+| [OnFinality](https://onfinality.io/networks/stellar) | ❌ | ❌ | ✅ | ❌ | ✅ | ❌ |
+| [Quasar](https://quasar.lightsail.network/) | ❌ | ❌ | ❌ | ❌ | ✅ | ✅ |
```

Comment:
I personally would remove the separate column for this in the table, and have it be another footnote option like the other two things below. If we add a column for this, why wouldn't we add a "Horizon Historical Access" column?